pkg/kubelet/cm/dra/manager_test.go

```diff
@@ -117,7 +117,8 @@ func setupFakeDRADriverGRPCServer(shouldTimeout bool) (fakeDRAServerInfo, error)
 		driverName: driverName,
 	}
 	if shouldTimeout {
-		timeout := plugin.PluginClientTimeout + time.Second
+		plugin.PluginClientTimeout = time.Millisecond * 10
+		timeout := plugin.PluginClientTimeout * 2
 		fakeDRADriverGRPCServer.timeout = &timeout
```

Comment:
Would it make sense to set `timeout := time.Millisecond * 20` ?

Reply:
We just need to make the timeout time greater than PluginClientTimeout, so I added 10ms on top of PluginClientTimeout.